components/payments/payment-modal.js

```diff
@@ -20,13 +21,21 @@ export default function AddPaymentModal({ showModal, setShowModal, addNewPayment
           label="Account Number"
           refEl={acctNumInput}
         />
+        <Input
+          id="expireDate"
+          type="month"
+          label="Expiration Date"
+          refEl={expireDateInput}
+        />
       </>
       <>
         <button
           className="button is-success"
           onClick={() => addNewPayment({
-            acctNumber: acctNumInput.current.value,
-            merchant: merchantNameInput.current.value
+            account_number: acctNumInput.current.value,
+            merchant_name: merchantNameInput.current.value,
+            expiration_date: `${expireDateInput.current.value}-01`,
```

Comment:
for me right now whatever year i have put in here, it posts 2024 regardless.

Reply:
Have the same problem as well, issue still persists.

Reply:
this is actually an api issue, where create date and expiration date were mixed up. will make a pull request for this small fix 👍